ner_v2/detectors/temporal/date/en/date_detection.py

```diff
@@ -147,51 +201,18 @@ def get_exact_date(self, date_list, original_list):
-        self._update_processed_text(original_list)
-        date_list, original_list = self._gregorian_advanced_day_month_year_format(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._day_month_format_for_arrival_departure(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._date_range_ddth_of_mmm_to_ddth(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._date_range_ddth_to_ddth_of_next_month(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._gregorian_day_with_ordinals_month_year_format(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._gregorian_advanced_year_month_day_format(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._gregorian_year_day_month_format(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._gregorian_month_day_with_ordinals_year_format(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._gregorian_day_month_format(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._gregorian_month_day_format(date_list, original_list)
-        self._update_processed_text(original_list)
-
-        date_list, original_list = self._day_after_tomorrow(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._date_days_after(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._date_days_later(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._day_before_yesterday(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._todays_date(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._tomorrows_date(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._yesterdays_date(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._day_in_next_week(date_list, original_list)
-        self._update_processed_text(original_list)
-        date_list, original_list = self._day_range_for_nth_week_month(date_list, original_list)
-        self._update_processed_text(original_list)
+        if self.country_code in self.country_date_detector_preferences:
```

Comment:
check if country code is not none

Reply:
It shouldn't matter as we are giving all country codes which are not in self.country_date_detector_preferences including 'None' the same detector preferences.